Content.Shared/Preferences/Loadouts/RoleLoadout.cs

```diff
@@ -255,7 +255,12 @@ public void SetDefault(HumanoidCharacterProfile? profile, ICommonSession? sessio
     /// <summary>
     /// Returns whether a loadout is valid or not.
     /// </summary>
-    public bool IsValid(HumanoidCharacterProfile profile, ICommonSession? session, ProtoId<LoadoutPrototype> loadout, IDependencyCollection collection, [NotNullWhen(false)] out FormattedMessage? reason)
+    public bool IsValid(
+        HumanoidCharacterProfile profile,
+        ICommonSession? session,
+        ProtoId<LoadoutPrototype> loadout,
+        IDependencyCollection collection,
+        [NotNullWhen(false)] out FormattedMessage? reason)
```

Comment:
This looks unchanged minus formatting
```suggestion
    public bool IsValid(HumanoidCharacterProfile profile, ICommonSession? session, ProtoId<LoadoutPrototype> loadout, IDependencyCollection collection, [NotNullWhen(false)] out FormattedMessage? reason)
```